### ISSUE TITLE
Incorrect default JDBC columns names

### ISSUE THREAD

**Issue by barteksc** [bucket4j/bucket4j#489]
Starting with `8.11`, it looks like there is a mistake when assigning default columns names in the `io.github.bucket4j.distributed.jdbc.AbstractJdbcProxyManagerBuilder` class:

```java
    private String idColumnName = "state";
    private String stateColumnName = "id";
```

These two columns names are swapped.

**Comment by vladimir-bukhtoyarov** [comment]
Hello @barteksc 

Thank you for reporting. I will deploy fix at weekend. Currently, just specify column names explicitly.

**Comment by vladimir-bukhtoyarov** [comment]
Fix released with version `8.12.1`
